srcpkgs/orocos-kdl/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'orocos-kdl'
+pkgname=orocos-kdl
+version=1.4.0
+revision=1
+#wrksrc=orocos_kinematics_dynamics-${version}/orocos_kdl/
+build_style=cmake
+makedepends="eigen3.2"
+short_desc="Orocos Kinematics and Dynamics C++ library"
+maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
+license="LGPL-2.1-only"
+homepage="http://www.orocos.org/kdl"
+distfiles="https://github.com/orocos/orocos_kinematics_dynamics/archive/v${version}.tar.gz"
+checksum=05b93e759923684dc07433ccae1e476d158d89b3c2be5079c20062406da7b4dd
+
+post_extract() {
+	mv ${XBPS_BUILDDIR}/orocos_kinematics_dynamics-${version}/orocos_kdl \
```

Comment:
This sounds like you want to set ```
wrksrc="orocos_kinematics_dynamics-${version}"
build_wrksrc="orocos_kdl"
```

Reply:
Ahh, I wasn't aware of that variable. I'll change it in all the templates.